### PR TITLE
Update toys relativeRotation/Position

### PR DESCRIPTION
**Pull request by Atlante45**
This updates the toys so that they are placed correctly in your hand when using the grab script.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: e894d56d342ee660d58361606d6d6402f189d18d available at: https://gist.github.com/d731deae3f9e82854bd9<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6593.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6593.dmg

File: interface/src/avatar/AvatarActionHold.cpp [hunk 1]
```diff
@@ -85,8 +85,8 @@ void AvatarActionHold::updateActionWorker(float deltaTimeStep) {
     QList<EntityActionPointer> holdActions = ownerEntity->getActionsOfType(ACTION_TYPE_HOLD);
     foreach (EntityActionPointer action, holdActions) {
         std::shared_ptr<AvatarActionHold> holdAction = std::static_pointer_cast<AvatarActionHold>(action);
-        glm::quat rotationForAction;
-        glm::vec3 positionForAction;
+        glm::quat rotationForAction { Quaternions::IDENTITY };
```
Comment (jherico) [inline interface/src/avatar/AvatarActionHold.cpp]: this is already the default value for a newly constructed quaternion.  Why the explicit initialization?

File: interface/src/avatar/AvatarActionHold.cpp [hunk 2]
```diff
@@ -85,8 +85,8 @@ void AvatarActionHold::updateActionWorker(float deltaTimeStep) {
...
+        glm::vec3 positionForAction { Vectors::ZERO };
```
Comment (jherico) [inline interface/src/avatar/AvatarActionHold.cpp]: same

File: interface/src/avatar/AvatarActionHold.cpp (hunk 1 above)
Comment (Atlante45) [inline interface/src/avatar/AvatarActionHold.cpp]: Sanity, had a bug that looked like this wasn't initialized correctly. Ended up staying, will remove.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5574f4736b368af38542b35c3cc370b72f39364a available at: https://gist.github.com/d731deae3f9e82854bd9<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/High+Fidelity-PR-6593-win32.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6593.dmg

**Pull request merged**